Makefile

```diff
@@ -1,10 +1,12 @@
 # # /bin/sh does not support source command needed in make test
 #SHELL := /bin/bash
 
-ROOT_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
-
+ROOT_DIR=$(shell git rev-parse --show-toplevel)
```

Comment:
Any specific need to change the old one?

Reply:
Much clearer and shorter, IMO